src/InputGroup/_InputGroupAddon.js

```diff
@@ -0,0 +1,45 @@
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const InputGroupAddon = ({
+    addonClassNames,
+    children,
+    className,
+    compact,
+    isButton,
+    ...otherProps
+}) => {
+
+    const addonClasses = classnames(
+        addonClassNames,
+        'fd-input-group__addon',
+        { 'fd-input-group__addon--button': isButton },
+        { 'fd-input-group__addon--button--compact': compact && isButton }
```

Comment:
The class `fd-input-group__addon--button--compact` doesn't exist so this can be removed.  See comment in `InputGroup.js` about how to handle `compact` prop.